renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+    "branchPrefix": "renovate-",
```

Comment:
We initially set this to `renovate-` because Jenkins did not like `renovate/`. Since we're going to probably be using Travis here, you may want to see if the build passes without setting this...